lib/plugins/aws/deploy/compile/events/apiGateway/lib/validate.js

```diff
@@ -4,80 +4,160 @@ const _ = require('lodash');
 
 module.exports = {
   validate() {
-    const httpEvents = [];
+    const events = [];
 
-    // validate that path and method exists for each http event in service
     _.forEach(this.serverless.service.functions, (functionObject, functionName) => {
-      functionObject.events.forEach(event => {
+      _.forEach(functionObject.events, (event) => {
```

Comment:
Same here, I think `functionObject.events.forEach(event => ...)` is totally fine.